packages/myst-directives/src/code.ts

```diff
@@ -200,6 +200,22 @@ export const codeCellDirective: DirectiveSpec = {
         ruleId: RuleId.directiveOptionsCorrect,
       });
     }
-    return [code];
+
+    const output = {
+      type: 'output',
+      id: nanoid(),
+      data: [],
+    };
+
+    const block = {
+      type: 'block',
+      meta: undefined, // do we need to attach metadata?
```

Comment:
This will come later during transformation, when labels, etc, are lifted out of code up to parent blocks.